.github/workflows/test.yml

```diff
@@ -101,14 +101,18 @@ jobs:
         with:
           target-ros2-distro: ${{ matrix.ros_distribution }}
           vcs-repo-file-url: "${{ github.workspace }}/.github/workflows/repo_file_for_test_cpp_package.repos"
-        # TODO(tmoulard): re-enable this test once it passes on Windows
-        if: runner.os != 'Windows'
       - run: test -d "${{ steps.test_all_packages_in_repo.outputs.ros-workspace-directory-name }}/install/osrf_testing_tools_cpp"
-        # TODO(tmoulard): re-enable this test once it passes on Windows
-        if: runner.os != 'Windows'
+        name: "Check that ament_copyright install directory is present (Non-Windows only)"
```

Comment:
If all of this `if windows do this; if !windows do that` just a switch on the path pattern?

If so, then this would be better as a `with:` directive to set the variable based on the matrix value. Is that possible? The duplicated lines are pretty verbose and hard to read, if we can get around it.

If it's not possible to do a conditional assignment in `with:`, then I think we might be better off writing a little test script instead of `run:` operations.

Reply:
`with` seems to only apply to composite actions, but i manged to do something similar with a fake ternary conditional expressions and job-level `env` properties